### PR TITLE
Order data objects on sample and presample endpoints

### PR DESCRIPTION
**Pull request by dblenkus**

### CODE REVIEW

**Comment by kostko** [comment]
Test are failing?

**Comment by mstajdohar** [comment]
Tests are still failing?

File: docs/CHANGELOG.rst [hunk 1]
```diff
@@ -58,6 +58,7 @@ Fixed
 * Fix ``htseq_count`` and ``featureCounts`` for large files
 * Fix ``upload gtf annotation``
 * Fix gene_id field type for differential expression storage objects
+* Order data objects in ``SampleViewSet`` and ``PresampleViewSet``
```
Comment (kostko) [inline docs/CHANGELOG.rst]: I think you reverted the change for presamples?

Comment (dblenkus) [inline docs/CHANGELOG.rst]: Thanks!

**Pull request merged**